### DISCUSSION TITLE
add icon pack to linux mint software manager

### DISCUSSION THREAD

**Discussion by RustoMCSpit** [SylEleuth/gruvbox-plus-icon-pack#54] [Ideas]
to do this, i believe you add it to the apt package manager and flathhub online

**Comment by SylEleuth** [comment]
In theory adding to the repo might not be that hard. But only in the theory. I'm on Arch and I would have to learn how to do that, write scripts and test it. It takes time and knowledge. Thankfully in that case I've had a help from other contributor who did that for me. I would love to have it in every distribution's repo but as I said I don't have knowledge nor time at the moment.
Even [Flathub](https://docs.flathub.org/docs/for-app-authors/requirements/) is a little bit more complicated than just adding it online.
